### PR TITLE
feat: updated codec versions

### PR DESCRIPTION
**Pull request by Ouwen**
Updates codecs to current version. 
Fixes this issue: #506

### CODE REVIEW

File: src/shared/decoders/decodeHTJ2K.js [hunk 1]
```diff
@@ -6,8 +6,8 @@
 // import openJphFactory from '../../../codecs/openjphjs.js';
 // import openjphWasm from '../../../codecs/openjphjs.wasm';
 
-import openJphFactory from '@cornerstonejs/codec-openjph/dist/openjphjs.js';
-import openjphWasm from '@cornerstonejs/codec-openjph/dist/openjphjs.wasm';
+import openJphFactory from '@cornerstonejs/codec-openjph/wasmjs';
```
Comment (sedghi) [inline src/shared/decoders/decodeHTJ2K.js:9]: :( I missed this one I guess to be consistent with others

**Comment by Ouwen** [comment]
@sedghi ready for review!

**Comment by sedghi** [comment]
Just to add to the comments above, we did rigorous testings and this PR does not add performance issues anymore

**Pull request merged**